packages/cozy-stack-client/src/FileCollection.js

```diff
@@ -53,6 +53,9 @@ class FileCollection extends DocumentCollection {
     this.specialDirectories = {}
   }
 
+  /**
+   * Get a file by id
```

Comment:
What is the return? ;)

Reply:
I think we should try to use the verb fetch when doing something remote and get only when it's local.

Reply:
I agree, that said the method is also called get.

Reply:
> I agree, that said the method is also called get.

So ? There are several options here : 

- add a new method with a new name, deprecate the other
- Create an issue not to forget
- Having the docstring clearer than the name can help

IMHO we should not stay with bad names.

Reply:
In general I think fetch instead of get is a clearer, especially in cozy-stack-client, though I'm less sure about doing the same thing in cozy-client.
The goal of this PR was only to make the existing `get` method appear in the docs, not to refactor anything, so I used `fetch` in the docstring without changing the code.